packages/camera/lib/camera.dart

```diff
@@ -1,355 +1,14 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+library camera;
```

Comment:
Instead of relying on a library-name you can use an import path in your part of statements:

```
part of 'camera.dart`;
```